### ISSUE TITLE
Label isn´t recognized by screenreader 

### ISSUE THREAD

**Issue by josuavogel** [jshjohnson/Choices#937]
**Describe the bug**
If someone uses a screenreader, he/she will get no information, which label is associated with the focused input.

**To Reproduce**
1. Active a screenreader (e.g. VoiceOver)
2. Tab through a form with choices select
3. No label is associated with the focused dropdown list

**Expected behavior**
The drowpdown should be connected with the form label. The screenreader should inform the user of the context of the drowdown.

**Desktop (please complete the following information):**
 - OS: MacOS Big Sur
 - Browser all

**Additional context**
Tested by using VoiceOver.
